dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -38,6 +39,7 @@ type Props = {
 	innerBackgroundColour?: string;
 	showDateHeader?: boolean;
 	editionId?: EditionId;
+	treats?: TreatType[];
```

Comment:
Is this ever optional as [the parent type](https://github.com/guardian/dotcom-rendering/pull/5557/files#diff-06ec888dd5f5b04e928175cbb9dbe5377633973349b1fc48b069b5fd49d66dbaR21) is required?

Reply:
`ContainerLayout` is used in lots of places outside of the fronts flow and in those cases - at this time - no treats are given